core/src/main/java/org/openstack4j/openstack/networking/internal/NetworkingServiceImpl.java

```diff
@@ -18,7 +18,7 @@
 
 /**
  * OpenStack Networking Operations API
- * 
+ *
```

Comment:
revert tab/whitespace changes in this file

Reply:
TenantId comes from Resource interface. I can change that, but that would entail a change on multiple classes outside of the scope for this enhancement. I renamed the field without renaming the getter.